mobile/src/screens/ProfileScreen.tsx

```diff
@@ -13,18 +14,86 @@ import { I18nContext } from '../context/I18nContext';
 import { Language, Languages } from '../models/Languages';
 import { AuthContext } from '../context/AuthContext';
 import { APIContext } from '../context/APIContext';
+import { Book } from '../models/Book';
+import { PaginatedBookList } from '../components/PaginatedBookList';
+import { useIsFocused } from '@react-navigation/native';
 
+const BOOKS_PER_PAGE = 9; 
+// or to simulate column book list, BOOKS_PER_PAGE = favorites.length + (3 - favorites.length % 3)
+
+/**
+ * Saved Tab to display user's favorited books
+ */
 const SavedTab: React.FC = () => {
+
+  const client = useContext(APIContext);
+  const auth = useContext(AuthContext);
+  const i18n = useContext(I18nContext);
+
+  // Favorite/saved books state
+  const [favorites, setFavorites] = useState<Book[]>([]);
+
+  // determines if the saved tab is unfocused/focused
+  const isFocused = useIsFocused();
+
+  // Rerender tab and get updated favorites when isFocused changes
+  // (Ensure that Profile AND Book Screens have synchronised favorites info)
+  useEffect(
+    () => {
+      (async () => {
+        try {
+          if (auth.user !== null) {
+            const res = await client.getFavorites();
+            setFavorites(res);
+          }
+        } catch (e) {
+          console.log(e.message);
+        }
+      })();
+    }, [isFocused]
+  );
+
+
   return (
     <View>
-      <Text>
-        Saved
-      </Text>
-      <View style={{ height: 300 }} />
+      {auth.user === null ? (
+        <View style={styles.unAuthenticatedContainer}>
+          <Text style={TextStyles.caption1}>{i18n.t("loginToSave")}</Text>
+          <View style={{ height: 300 }} />
+        </View>
+      )
+        :
+        (Array.isArray(favorites) && favorites.length > 0 ?
+          (
+            <View>
+              <PaginatedBookList books={favorites} booksPerPage={BOOKS_PER_PAGE} />
+            </View>
```

Comment:
Not a fan of reusing Paginated here, especially with the "Page 1 of 1" at the bottom. Patched ColumnBookList for you to remove the flatlist, sent it to you in slack